dashboard-app/src/styles/style_guide.ts

```diff
@@ -3,19 +3,24 @@
  */
 
 export enum ColoursEnum {
+  black = '#333333',
   white = '#FFFFFF',
   offWhite = '#F8F8F8',
   grey = '#707070',
-  light = '#DBDBDB',
+  light = '#B7B7B7',
+  orange = '#FFBF00',
   primary = '#8000FF',
   darkPrimary = '#5500aa',
   font = '#222222',
 }
 
 export enum FontSizeEnum {
-  heading = '1.6em',
-  heading2 = '1.2em',
-  medium = '0.8em',
+  heading = '2.75em',
```

Comment:
Can font sizes be amended in another PR? This drastically changes the designs of other pages.